### PR TITLE
Rasters merge

### PR DESCRIPTION
**Pull request by michaeldorman**
raster merge

### CODE REVIEW

**Review by Robinlovelace — COMMENTED** [review]
Looks good to me, have not had a chance to test but great work on keeping this moving.

File: 04-spatial-operations.qmd [hunk 1]
```diff
@@ -911,9 +911,50 @@ This returns the statistics for each category, here the mean elevation for each
 
 ### Merging rasters
 
-...
+Suppose we would like to compute the NDVI (see @sec-raster-local-operations), and additionally want to compute terrain attributes from elevation data for observations within a study area. 
+Such computations rely on remotely sensed information. 
+The corresponding imagery is often divided into scenes covering a specific spatial extent (i.e., "tiles"), and frequently, a study area covers more than one scene. 
+Then, we would need to merge (also known as "mosaic") the scenes covered by our study area.
```
Comment (Nowosad) [inline 04-spatial-operations.qmd:917]: @michaeldorman I would say that there is a slight difference between merge (gluing two non-overlapping images) and mosaic (some images overlap and we need to do something with the overlapped areas, e.g., calculate a mean). Would it be good to mention that?

Comment (michaeldorman) [reply 04-spatial-operations.qmd:917]: > @michaeldorman I would say that there is a slight difference between merge (gluing two non-overlapping images) and mosaic (some images overlap and we need to do something with the overlapped areas, e.g., calculate a mean). Would it be good to mention that?

I wasn't aware of this distinction, that's a good idea to mention but perhaps also acknowledge that it's not always reflected in function names. For example, software usually have one function for both cases and named either way (for example, R has `stars::st_mosaic` and Python has `rasterio.merge.merge`).

Comment (Nowosad) [reply 04-spatial-operations.qmd:917]: It would be triple check if my understanding is correct. But if it is -- I would suggest to add a note about it.

**Pull request merged**